docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -540,11 +540,19 @@ To migrate content from one collection to another, if the collections are parts
 
 #. Copy content from the source (old) collection to the target collection.
 #. Deprecate the module/plugin with ``removal_version`` scheduled for the next major version in ``meta/runtime.yml`` of the source collection. The deprecation must be released after the copied content has been included in a release of the target collection.
-#. When the next major release comes:
-
-  * remove the module/plugin from the source collection
-  * add ``redirect`` to the corresponding entry in ``meta/runtime.yml``
-  * remove ``removal_version`` from there
+#. When the next major release of the old collection is prepared:
+
+  * remove the module/plugin from the source collection and its symlink stored in ``plugin/modules`` directory if appropriate (mainly when removing from ``community.general`` and ``community.network``)
+  * remove its unit and integration tests, specific module utils, and documentation fragments if there are any in the source collection
+  * add a changelog fragment containing entries for ``removed_features`` and ``breaking_changes``; refer to the fragment in this `pull request <https://github.com/ansible-collections/community.general/pull/1304>`_ as an example
```

Comment:
I had to read this twice to figure out what it meant . . . not sure how to express this better, maybe:
```suggestion
  * add a changelog fragment containing entries for ``removed_features`` and ``breaking_changes``; you can see an example of a changelog fragment in this `pull request <https://github.com/ansible-collections/community.general/pull/1304>`_ 
```

Reply:
LGTM